client/pk-offline-update.c

```diff
@@ -32,7 +32,13 @@
 #include <packagekit-glib2/pk-offline-private.h>
 #include <stdlib.h>
 #include <unistd.h>
+#ifdef HAVE_SYSTEMD_SD_JOURNAL_H
 #include <systemd/sd-journal.h>
+#else
+#define LOG_INFO STDERR_FILENO
+#define LOG_WARNING STDERR_FILENO
+#define sd_journal_print dprintf
+#endif
```

Comment:
Does this actually work? This seems a bit sketchy...

Reply:
Yes, it prints correctly here on FreeBSD.

Reply:
Huh, okay...